### PR TITLE
Add Support for filtering by Country using components 

### PR DESCRIPTION
**Pull request by dzanfardino**
https://developers.google.com/maps/documentation/javascript/geocoding#GeocodingRequests

### CODE REVIEW

**Comment by freekmurze** [comment]
Could you also update the readme? Thanks!

**Comment by dzanfardino** [comment]
@freekmurze Done!

File: .idea/geocoder.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (freekmurze) [inline .idea/geocoder.iml]: Please remove the `.idea` directory from this commit.

Comment (dzanfardino) [reply .idea/geocoder.iml]: Sure thing. Sorry about that!

**Comment by freekmurze** [comment]
Thank you!

**Pull request merged**